filestore/handlers.py

```diff
@@ -3,8 +3,8 @@
 from .retrieve import HandlerBase
 from .readers.spe import PrincetonSPEFile
 from collections import deque
-import six
-from six.moves import range
+# import six
```

Comment:
I know, but the local practice is to import six at the top of every file
because it makes smoothing over the 2 v 3 issues easier.

On Sun, Mar 8, 2015 at 6:05 PM Stuart Wilkins <notifications@github.com>
wrote:

> In filestore/handlers.py
> <https://github.com/tacaswell/filestore/pull/1#discussion_r26009716>:
>
> > @@ -3,8 +3,8 @@
> >  from .retrieve import HandlerBase
> >  from .readers.spe import PrincetonSPEFile
> >  from collections import deque
> > -import six
> > -from six.moves import range
> > +# import six
>
> But it is not used? Best wishes, Stuart From: Thomas A Caswell <
> notifications@github.com<mailto:notifications@github.com>> Reply-To:
> tacaswell/filestore <reply@reply.github.com<mailto:reply@reply.github.com>>
> Date: Sunday, March 8, 2015 at 4:18 PM To: tacaswell/filestore <
> filestore@noreply.github.com<mailto:filestore@noreply.github.com>> Cc:
> Stuart Wilkins <swilkins@bnl.gov<mailto:swilkins@bnl.gov>> Subject: Re:
> [filestore] ENH : Added capability of returning 3D arrays (#1
> <https://github.com/tacaswell/filestore/pull/1>) In filestore/handlers.py<
> https://github.com/tacaswell/filestore/pull/1#discussion_r26008521>:
>  @@ -3,8 +3,8 @@ from .retrieve import HandlerBase from .readers.spe
> import PrincetonSPEFile from collections import deque -import six -from
> six.moves import range +# import six
> don't comment out six import please — Reply to this email directly or view
> it on GitHub<https://github.com/tacaswell/filestore/pull/1/files#r26008521
> >.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tacaswell/filestore/pull/1/files#r26009716>.
>